dags/looker.py

```diff
@@ -153,4 +153,33 @@
         **airflow_gke_prod_kwargs,
     )
 
-    lookml_generator_staging >> lookml_generator_prod
+    looker_folders_to_validate = [
+        706,  # KPI metrics
```

Comment:
This seems like the folder with the most important dashboards for now. Validation took about 3 minutes when I tested locally. We can add more folders as we test this out more